source/_components/sensor.zabbix.markdown

```diff
@@ -1,8 +1,8 @@
 ---
 layout: page
 title: "Zabbix Sensor"
-description: "Instructions on how to integrate Zabbix Triggers sensors within Home Assistant."
-date: 2016-12-13 22:57
+description: "Instructions on how to integrate Zabbix Triggers and Item sensors within Home Assistant."
+date: 2020-01-20 11:57
```

Comment:
Please do not change the date

Reply:
Actually it should be removed 😉

Reply:
As requested, I've added a patch to remove the date